extension/lib/BiasDoorhanger.jsm

```diff
@@ -137,7 +139,15 @@ class BiasDoorhanger {
       const timeSinceShown = Date.now() - stats.timestamp;
       const shouldShow = !stats.neverAgain && timeSinceShown > Config.showDoorhangerInterval;
 
-      if (hostname && isTreatmentPhase && isTracked && shouldShow) {
+      const branchName = Services.prefs.getCharPref(STUDY_BRANCH_PREF, "");
+      let branch = Config.branches.find(b => b.name === branchName);
+      if (!branch) {
+        branch = await Pioneer.utils.chooseBranch();
```

Comment:
I'm getting this error from `PioneerUtils.jsm`:
```
Error: Assertion error, Did not chose a value
```

Reply:
This is a string interpolation error in PioneerUtils